### ISSUE TITLE
Cannot update icon props

### ISSUE THREAD

**Issue by VonD** [corymsmith/react-native-icons#8]
Hi,
First of all many thanks for this great component!
I don't seem to be able to change icon props once it's been drawn : 

```
<Icon
    size={ 20 }
    name="ion|beer"
    color={ (this.state.selected && 'red') || 'blue' }
    style={ {height: 20, width: 20} }
/>
```

The icon is correctly drawn on first render, but the color doesn't change on state update. Am I doing something wrong ?

Many thanks

**Comment by corymsmith** [comment]
@VonD Glad you like it! That looks like a bug to me, I'll have to dig into it

**Comment by PhilippKrone** [comment]
Hello,

I just stumbled across the same thing - I would like to change the color of an icon during runtime (i.e. after first render) and it is just not happening. :)

Thanks!
Philipp

**Comment by corymsmith** [comment]
@VonD @PhilippKrone This was just fixed by @badfortrains. I've merged it in and published 0.0.3 to npm.
